R/save_tt.R

```diff
@@ -51,24 +51,24 @@ save_tt <- function(x, output, overwrite = FALSE) {
   file_ext <- tools::file_ext(output)
 
   output_format <- switch(file_ext,
-  "png" = "html",
-  "html" = "html",
-  "pdf" = "latex",
-  "tex" = "latex",
-  "md" = "markdown",
-  "Rmd" = "markdown",
-  "qmd" = "markdown",
-  "txt" = "markdown",
-  "docx" = "markdown",
-  "typ" = "typst",
-  stop("The supported file extensions are: .png, .html, .pdf, .tex, and .md.", call. = FALSE))
+    "png" = "html",
+    "html" = "html",
+    "pdf" = "latex",
+    "tex" = "latex",
+    "md" = "markdown",
+    "Rmd" = "markdown",
+    "qmd" = "markdown",
+    "txt" = "markdown",
+    "docx" = "markdown",
+    "typ" = "typst",
+    stop("The supported file extensions are: .png, .html, .pdf, .tex, and .md.", call. = FALSE)
```

Comment:
I think you forgot some extensions in the error message compared to choices in `switch()`